gnd/src/main/java/com/google/android/gnd/ui/home/HomeScreenViewModel.java

```diff
@@ -40,6 +44,7 @@
   private final DataRepository dataRepository;
   private final Navigator navigator;
   private final LiveData<Resource<Project>> activeProject;
+  private final BehaviorProcessor<Feature> featureProcessor;
```

Comment:
You're right about that. But you're even more right about making this a Subject instead :). So I'll refrain from this in favor of making this a subject. This is a good pointer though! Behaviors and Publishers are effectively the same here, as you said, since we subscribe in the constructor (so we're on board from the beginning of the stream), but anywhere else using the wrong one could lead to unfortunate repetitions. The Behavior will push the last emitted value to new subscribers while the Publisher won't. This would be bad here as if we didn't subscribe in the constructor it'd result in adding some feature twice. yikes